Resources/Textures/Objects/Specific/Hydroponics/holymelon.rsi/meta.json

```
@@ -1,7 +1,7 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from https://github.com/vgstation-coders/vgstation13/commit/b459ea3fdee965bdc3e93e7983ad7fa610d05c12 and https://github.com/tgstation/tgstation/commit/ead6d8d59753ef033efdfad17f337df268038ff3 and modified by slarticodefast",
+  "copyright": "Taken from https://github.com/vgstation-coders/vgstation13/commit/b459ea3fdee965bdc3e93e7983ad7fa610d05c12 and https://github.com/tgstation/tgstation/commit/ead6d8d59753ef033efdfad17f337df268038ff3 and modified by slarticodefast, Holy melon produce sprite modified by Chaoticaa (GitHub), all growth stage sprites and harvest sprite created by Chaoticaa (GitHub)",
```

```suggestion
  "copyright": "Taken from https://github.com/vgstation-coders/vgstation13/commit/b459ea3fdee965bdc3e93e7983ad7fa610d05c12 and https://github.com/tgstation/tgstation/commit/ead6d8d59753ef033efdfad17f337df268038ff3 and modified by slarticodefast, produce sprite modified by Chaoticaa (GitHub), all growth stage, harvest, and dead sprites created by Chaoticaa (GitHub)",
```

Note this is applicable if you're the sole creator of the dead sprite. If you modified it you have to specify.